change/@ni-nimble-components-b897b4de-cdc6-4a5c-84f7-e4a2fc2c9c07.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
While testing I noticed a small bug. It appears to be pre-existing so no need to fix in this PR.

1. filter mode is 'none'
2. tab focus the select
3. without opening, type 'z' and Zurich is correctly selected with a change event fired
4. tab away
5. shift tab back
6. Enter to open, arrow keys to change the value, Enter to commit
7. Now type 'z' again. Nothing happens.

LMK if you want a bug for this.

Reply:
Yes, please file a bug.

Reply:
https://github.com/ni/nimble/issues/2082